### ISSUE TITLE
Lags when creating chunks

### ISSUE THREAD

**Issue by Kamimimi** [BradFitz66/Stylized-Terrain-URP#1]
hey thank you so much for the open source! i tried to install it and it kinda works but its lagging hard on my pc when im creating chunks. i have a rtx 4080 and 64gb ram so unity shouldnt be an issue but my concern is that unity edit mode is probably only using my cpu rather than gpu, im not 100% sure though..? didnt you have that problem and also, do you know how i can make unity edit mode use my gpu instead?

**Comment by BradFitz66** [comment]
Yeah I've noticed some issues with creating chunks, resulting in the GPU timing out or crashing. Will look into this.
